synth.py

```diff
@@ -44,11 +44,26 @@
 service = 'datastore'
 versions = ['v1']
 
-for version in versions:
+protos = [
+  {
+    'service': 'datastore',
+    'version': 'v1',
+  },
+  {
+    'service': 'datastore-admin',
+    'version': 'v1',
+  }
+]
```

Comment:
I suppose you can exclude `version` field here and in line 60 just use a `"v1"` string, as they are similar in both services

Reply:
It's for the future use. They use to have an array of one element:  
`versions = ['v1'] ` to iterate over, so I would prefer to keep this.